Sources/IBLinterKit/Rules/UseBaseClassRule.swift

```diff
@@ -37,7 +37,7 @@ extension Rules {
             let violation: [Violation] = {
                 guard let baseClassesForElement = baseClasses[view.elementClass] else { return [] }
                 guard let customClass = view.customClass else {
-                    let message = "CustomClass is not set to \(viewName(of: view))"
+                    let message = "You must use a subclass of \(viewName(of: view))"
```

Comment:
I can also make this part into a separate PR if you prefer. I was a little confused when I ran the linter and saw this error since it says something like "CustomClass is not set to UILabel" even though we actually want the developer to set the custom class to something other than UILabel. Open to suggestions / improvements!